tests/integrations/tests/test_anthropic.py

```
@@ -846,39 +880,566 @@ def test_16_extended_thinking_streaming(self, anthropic_client, test_config, pro
+                )
+
+            # Delete the file
+            response = client.beta.files.delete(uploaded_file.id)
+
+            # Validate response - providers may return different formats
+            assert response is not None, "Delete response should not be None"
+
+            print(f"Success: Deleted file {uploaded_file.id} (provider: {provider})")
+
+            # Verify file is no longer retrievable
+            with pytest.raises(Exception):
```

Ruff (0.14.8)

Do not assert blind exception: `Exception`

(B017)